demos/02-2d-tsunami.ipynb

```diff
@@ -439,10 +460,13 @@
     "fig, axes  = plt.subplots(nrows, 2, figsize=(16, 6*nrows))\n",
     "\n",
     "for idx, ax in enumerate(axes.flatten()):\n",
-    "    filename =  os.path.join(options.output_directory, 'hdf5','Elevation2d_%05d' % idx)\n",
-    "    dc = DumbCheckpoint(filename, mode=FILE_READ)\n",
-    "    dc.load(elev)\n",
+    "    filename =  os.path.join(options.output_directory, 'hdf5','Elevation2d_%05d.h5' % idx)\n",
+    "    \n",
+    "    dc = CheckpointFile(filename, mode=FILE_READ)\n",
+    "    test = dc.load_mesh()\n",
+    "    elev = dc.load_function(test, 'elev_2d')\n",
     "    dc.close()\n",
```

Comment:
Perhaps use a variable name other than `dc`? Since we are no longer referring to `DumbCheckpoint`.